<launch>

	<arg name="port" />
	<node name="gps_rtk_driver" pkg="gps_driver" type="rtk_driver.py" args="$(arg port)" />

</launch>
